kytos/cli/commands/napps/parser.py

```diff
@@ -57,15 +58,21 @@ def call(subcommand, args):
 
 
 def parse_napps(napp_args):
-    """Return a list of username and napp_name from the napp list argument.
+    """Return a list of tuples with username, napp_name and version.
 
-    The expected format of a NApp is napp_username/napp_name.
+    Each napp arg must to have the pattern username/name-version. Version is
+    optional, whether no version is found the tuple will be
+    ('username', 'name', None).
```

Comment:
Let's try to improve readability here.
I propose renaming `napp_args` to `napp_ids` or `list_of_napp_ids` [It is important to be consistent across all projects with naming!].

And I also agree with @cemsbr text proposal.